providers/wpar.rb

```diff
@@ -17,7 +17,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 require 'chef/mixin/shell_out'
-require 'wpars'
+begin
+  require 'wpars'
+rescue LoadError # rubocop:disable Lint/HandleExceptions
+  # This space left intentionally blank.
```

Comment:
Yes but this is minimum-viable-don't-break-shit as a fix first. Because the gem is needed in the `load_current_resource` it makes it a bit more annoying to fix properly :)